website/pages/docs/secrets/databases/mysql-maria.mdx

```diff
@@ -84,8 +84,8 @@ the proper permission, it can generate credentials.
     lease_id           database/creds/my-role/2f6a614c-4aa2-7b19-24b9-ad944a8d4de6
     lease_duration     1h
     lease_renewable    true
-    password           8cab931c-d62e-a73d-60d3-5ee85139cd66
-    username           v-vaultuser-e2978cd0-
+    password           yY-57n3X5UQhxnmFRP3f
+    username           v_vaultuser_my-role_crBWVqVh2Hc1
```

Comment:
Should it only be underscores?
```suggestion
    username           v_vaultuser_my_role_crBWVqVh2Hc1
```

Reply:
No because the role name is `my-role` and there isn't an explicit change from `-` to `_`